src/Faithlife.WebRequests/Json/AutoWebServiceRequest.cs

```diff
@@ -67,13 +67,28 @@ protected virtual TResponse CreateResponse()
 			bool isStatusCodeHandled = false;
 			HttpStatusCode statusCode = webResponse.StatusCode;
 			string statusCodeText = statusCode.ToString();
-			PropertyInfo statusCodeProperty = GetProperty(responseType, statusCodeText);
+			PropertyInfo resultProperty = GetProperty(responseType, statusCodeText);
 			object content = null;
+
+			if (resultProperty != null && resultProperty.CanWrite)
+			{
+				Type resultPropertyType = resultProperty.PropertyType;
+				content = await ReadContentAsAsync(info, resultPropertyType).ConfigureAwait(false);
+				resultProperty.SetValue(response, content, null);
+				isStatusCodeHandled = true;
+			}
+
+			PropertyInfo statusCodeProperty = GetProperty(responseType, nameof(HttpResponseMessage.StatusCode));
```

Comment:
I see no value in using `nameof(HttpResponseMessage.StatusCode)` here; using `"StatusCode"` would make it easier to understand what's happening, IMHO.

Reply:
Updated. 9c6a8a47ed60beacfa362fcb6831fa078b832668